### ISSUE TITLE
gmailctl.libsonnet update

### ISSUE THREAD

**Issue by legeana** [mbrt/gmailctl#77]
When the user runs `gmailctl init` the `gmailctl.libsonnet` is created. Unfortunately the file it never updated automatically again.
I propose `gmailctl init --reset-lib` or similar flag to force update all library files (current and future).
It is understandable that `gmailctl` will not rewrite `config.jsonnet` as it may contain user configuration, however library files are not expected to be touched by the user.

**Comment by mbrt** [comment]
Sounds reasonable!

**Comment by paravoid** [comment]
Why is the library copied in the first place though? If the user wants to override these functions, they can always copy it themselves and import it separately in their own configuration. Basically, I view `gmailctl.libsonnet` as gmailctl's stdlib, which can be expanded, updated, bugs fixed etc. and it'd all transparent to the user's config. Is that unreasonable?

**Comment by mbrt** [comment]
> If the user wants to override these functions, they can always copy it themselves and import it separately in their own configuration.

Correct, however when this was implemented, I kept two things in mind:

* The user will see the library on the side of their config, so if they have any doubt they can take a look at it easily, without having to look at the source code here.
* Binary updates don't affect older installations, so as long as you keep the standard library you are using along with the config, you won't be affected even if new versions are released. This frees me from guaranteeing 100% compatibility for new library releases.

I think the way @legeana put it down is the safest.
